### PR TITLE
feat: docker configs

### PR DESCRIPTION
**Pull request by amanji**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from pydantic_settings import BaseSettings
+
+
+class Settings(BaseSettings):
+    aries_vcr_url: str = "http://host.docker.internal:8080"
```
Comment (esune) [inline config.py:5]: This value is fine for local development, but we'll need to override it when deploying the solution. Can we use an environment valuable to set it, and fall-back on this if nothing is specified?

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+FROM python:3.12
+
+WORKDIR /app
+
+ENV VIRTUAL_ENV=/opt/venv
+RUN python3 -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
```
Comment (esune) [inline docker/Dockerfile:7]: Do we need a virtual environment created when using poetry and inside a container? This bit might be unnecessary. Example poetry Dockerfile: https://github.com/hyperledger/aries-endorser-service/blob/main/endorser/Dockerfile.endorser

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+services:
+  aries-vcr-vc-service:
+    build:
+     context: ..
+     dockerfile: docker/Dockerfile
+    environment:
+      ARIES_VCR_URL: "http://vcr-api:8080"
```
Comment (esune) [inline docker/docker-compose.yml:7]: This does not appear to be read anywhere - related to the comment in `config.py`.

**Review by esune — CHANGES_REQUESTED** [review]

File: config.py (hunk 1 above)
Comment (amanji) [reply config.py:5]: `pydantic-settings` automatically pulls this value from the environment. Setting `ARIES_VCR_URL=<url>` will set this accordingly. The string there is just a fallback.

File: docker/Dockerfile (hunk 2 above)
Comment (amanji) [reply docker/Dockerfile:7]: Not sure but I can try running it without that set

**Review by esune — APPROVED** [review]

**Pull request merged**